provider.go

```diff
@@ -74,15 +166,28 @@ func getAPI(hostAddr string, maxRetries int) (client.API, error) {
 func Provider() terraform.ResourceProvider {
 	return &schema.Provider{
 		Schema: map[string]*schema.Schema{
-			"tunnel_address": &schema.Schema{
+			"driver": &schema.Schema{
 				Type:     schema.TypeString,
-				Required: true,
+				Optional: true,
+				Default: "tunnel",
+				Description: "Driver to use to connect to Fleet. Can be tunnel or api.",
```

Comment:
Heh, I was trying to do hacktoberfest too. Open as many PRs as you want, as long as each one has a more or less coherent purpose.